### PR TITLE
Feature/6.9 6.21 40112315

### PR DESCRIPTION
**Pull request by muhadovic**
https://freedcamp.com/dashboard/home/panel/task/40590174

Updated Screens [Screenshots]
- 6.9 Likes List
image

- 6.11 Change Status Modal
image
- 6.12 Change Status Auction
image
- 6.14 deal Approved Modal
image
- 6.18 Bit_live_auction
image
image

- 6.19 Bid Buy
image

- 6.15 Status in Review
image

### CODE REVIEW

File: src/features/nft/nftModals/likeListData.ts [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-  { id: 11, date: '9/11/21', userName: 'Z TestName', sold: 9, followers: 9 },
-  { id: 12, date: '8/11/21', userName: 'O TestName', sold: 8, followers: 1 },
-  { id: 13, date: '7/11/21', userName: 'N TestName', sold: 7, followers: 4 },
-  { id: 14, date: '6/11/21', userName: 'R TestName', sold: 3, followers: 3 },
-  { id: 15, date: '5/11/21', userName: 'X TestName', sold: 11, followers: 5 },
-  { id: 16, date: '4/11/21', userName: 'B TestName', sold: 6, followers: 1 },
-  { id: 17, date: '3/11/21', userName: 'V TestName', sold: 8, followers: 2 },
-  { id: 18, date: '2/11/21', userName: 'I TestName', sold: 4, followers: 2 },
-  { id: 19, date: '1/11/21', userName: 'E TestName', sold: 7, followers: 3 },
+  { id: 0, date: '1 min ago', member: 'L TestName', sold: 1, followers: 4 },
```
Comment (ngvtuan) [inline src/features/nft/nftModals/likeListData.ts:2]: For `date` field, I think we can use `datetime` and use `dayjs ` to show. Refer https://day.js.org/docs/en/display/from-now

**Review by ngvtuan — COMMENTED** [review]
ui issue

File: src/common/components/ProgressBar/ProgressBar.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  const classes = cn(
+    'bg-success-pressed text-xs leading-none py-px h-full flex items-center justify-end rounded-2xl striped',
+  )
+
+  return (
+    <div className={cn('w-100 h-5', className)}>
+      <div className='shadow w-full h-full bg-gray-57 bg-opacity-20 rounded-2xl'>
+        <div
+          className={classes}
+          style={{ width: percent + '%', backgroundColor: '#00B282' }}
```
[the remark this answers is not in the document]
Comment (muhadovic) [reply src/common/components/ProgressBar/ProgressBar.tsx:20]: This progress bar is the striped progress bar. So it needs to use the background and backgroundColor properties at the same time. If backgroundColor would be used with TailwindCSS class, it doesn't work. So I used the backgroundColor property. If you have a good solution, please let me know

**Review by ngvtuan — APPROVED** [review]

**Review by goal-shevchenko — CHANGES_REQUESTED** [review]
All looks good to me except for one changes.

**Review by goal-shevchenko — APPROVED** [review]

**Pull request merged**